### ISSUE TITLE
MC.Eternal Crash on startup

### ISSUE THREAD

**Issue by JJPederson** [adam9899/Issues#221]
Hi there, my MC.Eternal crashed upon startup, loading it up from GDLauncher, it gave me the following message:
 

"An error during startup prevented Minecraft from starting
The following mods have been identified as potential causes:

Tinkers' Construct, Construct's Armory, Just Enough Items

It gave me a crash report, which can be found here: https://paste.dimdev.org/ozayigaqas.mccrash

Any help would be super greatly appreciated! ❤️

If it helps, I've allocated 4GB of ram. But have played ATM3 and 4 with no issues at all.

**Comment by Avalonzo** [comment]
I'm no expert at reading crash logs and I did a quick google search and looks like your running out of memory also copied and pasted from main modpack page - Recommended RAM: 5.5gb+ (8gb for maximum performance)

Please feel free to wait for advice from someone with a bit experience regarding crash logs :D

**Comment by pattplayz** [comment]
Yep, @Avalonzo is right. Seem to be running out of memory. You'll need to allocate more.
